extensions/common/sql/sql-lease/src/main/java/org/eclipse/edc/sql/lease/SqlLeaseContext.java

```diff
@@ -75,29 +78,12 @@ public void breakLease(String entityId) {
     public void acquireLease(String entityId) {
```

Comment:
we could take the opportunity to return a `StoreResult` that could fail with `AlreadyLeased` reason eventually, that would avoid a lot of try...catch by callers

Reply:
Done in https://github.com/eclipse-edc/Connector/pull/5207/commits/4ea6f3e02dde5e40029518b7e5bffc340aecdbe1